docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -77,16 +77,102 @@ Note that we use the `env` variable in the above example, which corresponds to t
 [discrete]
 [[synthetics-cli-params]]
 // lint ignore params
-== Using CLI arguments to set params
+== Use CLI arguments to set params
 
 To set parameters when running `elastic-synthetics` on the command line, use the `--params` or `-p` flag to the `elastic-synthetics` program. The provided map is merged over any existing variables defined in the `synthetics.config.{js,ts}` file.
 
 To override the `url` parameter, you would run: `elastic-synthetics . --params '{"url": "http://localhost:8080"}'`
 
+[discrete]
+[[synthetics-request-param]]
+== Use the `request` parameter
+
+Use the `request` parameter to make API requests independently of the browser interactions.
+
+// Link to Playwright docs?
+// https://playwright.dev/docs/test-api-testing#sending-api-requests-from-ui-tests
+
+NOTE: The `request` parameter is not intended to be used for writing pure API tests. Instead, it is a way to support
+writing plain HTTP requests in service of a browser-based test.
```

Comment:
Not sure I fully understood the meaning of this.

Was thinking something like
> writing plain HTTP requests that are separate to the browser-based test.

Reply:
This was derived from https://github.com/elastic/synthetics/pull/555.

Reply:
@andrewvc was this what you intended (it wasn't clear to me)?

Reply:
Confirmed https://github.com/elastic/observability-docs/pull/2071#discussion_r949526167.